docs/quickstart.rst

```diff
@@ -26,30 +26,38 @@ To check if your endpoint/client have network access and can connect to the func
 
   >>> curl https://api2.funcx.org/v2/version
 
-This should return a version string, for example: ``"0.3.0"``
+This should return a version string, for example: ``"1.0.2"``
 
 .. note:: The funcx client is supported on MacOS, Linux, and Windows. The funcx-endpoint
-   is only supported on Linux.
+   is only supported on Linux and MacOS.
```

Comment:
Do we officially support macOS now? I thought the official line was that it works, but only in a Docker container

Reply:
Taking out MacOS unless dev thread concludes otherwise.